Makefile

```diff
@@ -1,16 +1,8 @@
-.PHONY: gen dev-reset all clean test web website
+.PHONY: gen dev-reset all clean test web-$@ website-$@
 
-.SHELLFLAGS += ${SHELLFLAGS} -e
-PWD = $(shell pwd)
-UID = $(shell id -u)
-GID = $(shell id -g)
-NPM_VERSION = $(shell python -m scripts.npm_version)
-PY_SOURCES = authentik tests scripts lifecycle
-DOCKER_IMAGE ?= "authentik:test"
+all: lint-fix lint test gen web-all website-all  ## Lint, build, and test everything
 
-pg_user := $(shell python -m authentik.lib.config postgresql.user 2>/dev/null)
-pg_host := $(shell python -m authentik.lib.config postgresql.host 2>/dev/null)
-pg_name := $(shell python -m authentik.lib.config postgresql.name 2>/dev/null)
+include scripts/common.mk
 
 CODESPELL_ARGS = -D - -D .github/codespell-dictionary.txt \
```

Comment:
just a nitpicking: is the reason for not putting this into `scripts/common.mk` that it addresses the directories like `.github` and `website`?